rpc/core/consensus.go

```diff
@@ -63,9 +70,37 @@ func Validators(ctx *rpctypes.Context, heightPtr *int64) (*ctypes.ResultValidato
 	if err != nil {
 		return nil, err
 	}
+	// page & perPage === 0 then all validators are returned, no pagination
+	if page == 0 && perPage == 0 {
+		return &ctypes.ResultValidators{
+			BlockHeight: height,
+			Validators:  validators.Validators,
+		}, nil
+	}
+
+	totalCount := len(validators.Validators)
+	perPage = validatePerPage(perPage)
+	page, err = validatePage(page, perPage, totalCount)
+	if err != nil {
+		return nil, err
+	}
+	skipCount := validateSkipCount(page, perPage)
+	apiResults := make([]*types.Validator, cmn.MinInt(perPage, totalCount-skipCount))
```

Comment:
can't we just return `validators.Validators[skipCount...skipCount+cmn.MinInt(perPage, totalCount-skipCount)]`?